highfidelity/hifi
# Avoid the spot lights flickering in Home

- THe flickering comes from the transition between the Inside / outside volume of the spot light. We use 2 different techniques depending of the region in which is the eye to render the spot light.
in master, this transition fails when looking at the spot light origin from under (in the cone), we then are in a zone which is  OUTSIDE but the technique used for outside is failing there.

- To fix this quickly, i simply extended the zone considered  "INSIDE", this avoid to go through the problematic region.

- REfreshed some of the tools we use for monitoring rendering and put them in utilities / render to avoid one more folder level (they were in utilities/tools/render)


TO TEST:
Go in forum at the entrance of the dressing room.
Make sure the spot lights are turned on (switch against the wall to your left)
now translate toward the wall back and forth.
In master, you should see the spot lights flickering
not with this build
